### DISCUSSION TITLE
Moved files not accurately displayed in PR

### DISCUSSION THREAD

**Discussion by mpmarven-ms** [community/community#27638] [Pull Requests]
I just created a PR where I copied files from one project to another.  Effectively removing them from one project and adding them to another.  In the PR the only indications of this are the files in the final destination are tagged as renamed/edited.  While this is technically accurate I feel like this isn't entirely accurate, I would suggest a filter/toggle to reflect the change both ways, as it is now and optionally showing the files as removed from one project and added to another.  I have two use cases for this:

1.  When reviewing a PR I feel like this almost hides the move from the reviewer and if it wasn't hidden they might question why a file was moved from one project to another.
2.  The reason I even noticed this was because I was trying to look at the PR to see which files I had added/removed to double check that I removed everything from the original project.  But I can't really tell that easily with the currently way it's displayed.

**Comment by DmitriyShum** [comment]
You could possibly start a discussion on the repository about why the files were moved. The owner is able to see the exact changes and will be able to keep track of them.
